cmd/kots/cli/install.go

```diff
@@ -173,6 +173,15 @@ func InstallCmd() *cobra.Command {
 				}
 			}
 
+			isKurl, err := kotsadm.IsKurl(kubernetesConfigFlags)
+			if err != nil {
+				return errors.Wrap(err, "failed to check kURL")
```

Comment:
This will fail if the user's local kubeconfig doesn't have permission to list nodes. Does this break the minimalRBAC flag use?

Reply:
I updated this function to not error if we can't list nodes